.github/workflows/wasm_publish.yml

```diff
@@ -0,0 +1,61 @@
+name: Manual NPM Package Publish
+
+on:
```

Comment:
the ci should only trigger when creating a new tag on appflowy cloud

Reply:
This CI is manual, and it is not appropriate to bind the tag of this repo?